libtransact/Cargo.toml

```diff
@@ -110,6 +110,7 @@ redis-db = ["redis"]
 sawtooth-compat = ["sawtooth-sdk"]
 sqlite-db = ["rusqlite", "r2d2", "r2d2_sqlite"]
 state-merkle = ["cbor-codec"]
+state-merkle-redis-db-tests = ["redis-db", "state-merkle"]
```

Comment:
Maybe put a comment on this so its clear this is one one that you need to specify to run full redis tests?

Reply:
added

Reply:
typo - "merlke"

Reply:
Fixd